devops/cloudbuild-pr.yaml

```diff
@@ -16,12 +16,14 @@ steps:
   - name: node:20.5.1
     entrypoint: npm
     env:
+      - 'VUE_APP_AUTH_API_KEY=$_VUE_APP_AUTH_API_KEY'
       - 'VUE_APP_ADDRESS_COMPLETE_KEY=$_VUE_APP_ADDRESS_COMPLETE_KEY'
       - 'VUE_APP_BUSINESS_API_KEY=$_VUE_APP_BUSINESS_API_KEY'
       - 'VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID=$_VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID'
       - 'VUE_APP_GENESYS_ENV=$_VUE_APP_GENESYS_ENV'
       - 'VUE_APP_GENESYS_URL=$_VUE_APP_GENESYS_URL'
       - 'VUE_APP_GENESYS_ID=$_VUE_APP_GENESYS_ID'
+      - 'VUE_APP_PAY_API_KEY=$_VUE_APP_PAY_API_KEY'
```

Comment:
I added these keys (with values) to the business-create-ci-pr trigger config in GCP.

This will have to be done for the other UIs as well.